src/AutoRest.CSharp/Common/Input/Source/SourceMemberMapping.cs

```diff
@@ -1,20 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using Azure.Core;
-using Microsoft.CodeAnalysis;
-
-namespace AutoRest.CSharp.Input.Source
-{
-    public class SourceMemberMapping
-    {
-        public SourceMemberMapping(string originalName, ISymbol existingMember)
-        {
-            OriginalName = originalName;
-            ExistingMember = existingMember;
-        }
-
-        public string OriginalName { get; }
-        public ISymbol ExistingMember { get; }
-    }
```

Comment:
This type is not removed, I renamed this to `SourcePropertyMapping` and changed it into a `record` because apperantly this only applies to properties.